src/index.tsx

```diff
@@ -13,7 +12,7 @@ const rootNode = ReactDOM.createRoot(document.getElementById('root') as HTMLElem
 
 rootNode.render(
   <React.StrictMode>
-    <Provider store={store}>
+    <Provider>
```

Comment:
**Suggestion:** The Provider component requires a store prop. Add the store prop to the Provider component.
```suggestion
    <Provider store={store}>
```